03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p><code>GaussianVariant.__call__</code> should use  the seeded <code>rng</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>